tesseract_process_managers/src/core/process_planning_server.cpp

```diff
@@ -144,56 +144,75 @@ std::vector<std::string> ProcessPlanningServer::getAvailableProcessPlanners() co
 
 ProcessPlanningFuture ProcessPlanningServer::run(const ProcessPlanningRequest& request) const
 {
-  CONSOLE_BRIDGE_logInform("Tesseract Planning Server Received Request!");
-  ProcessPlanningFuture response;
-  response.problem->name = request.name;
-  response.problem->plan_profile_remapping =
-      std::make_unique<const PlannerProfileRemapping>(request.plan_profile_remapping);
-  response.problem->composite_profile_remapping =
+  CONSOLE_BRIDGE_logDebug("Tesseract Planning Server Received Request!");
+
+  auto it = process_planners_.find(request.name);
```

Comment:
Does this need to be here? This check is also in `ProcessPlanningServer::run(ProcessPlanningProblem::Ptr problem, ...` so it should get checked before execution. This will check it twice.

Reply:
Yea, probably can be removed.